js/contentForeground/contentWeVoteUI.js

```diff
@@ -318,22 +318,35 @@ function topMenu () {
   let topMarkup = '' +
     '<div id="topMenuContainer" class="topMenuContainer">' +
     '  <img id="orgLogo" class="gridOrgIcon" src="https://wevote.us/img/endorsement-extension/endorsement-icon48.png" alt="">' +
-    '  <div id="orgName" class="gridOrgName core-text"></div>' +
-    '  <span class="gridSend">' +
-    '    <span class="innerGridSend core-text">' +
-    '      <span class="topCommentLabel core-text">Send us a comment about this page: </span>' +
-    '      <input type="text" id="emailWe" class="core-text" name="email" placeholder="Your email" >' +
-    '      <input type="text" id="topComment" class="core-text" name="topComment" placeholder="Comment here..." >' +
-    '      <button type="button" id="sendTopComment" class="sendTopComment weButton u2i-button u2i-widget u2i-corner-all removeContentStyles">Send</button>' +
-    '    </span>' +
+    '  <span id="orgName" class="gridOrgName core-text"></span>' +
+    '  <button type="button" id="openAdminButton" class="openInAdminApp weButton u2i-button u2i-widget u2i-corner-all removeContentStyles">ADMIN APP</button>' +
+    '  <span class="innerGridSend core-text">' +
+    '    <span class="topCommentLabel core-text">Send us a comment about this page: </span>' +
+    '    <input type="text" id="emailWe" class="core-text" name="email" placeholder="Your email" >' +
+    '    <input type="text" id="topComment" class="core-text" name="topComment" placeholder="Comment here..." >' +
+    '    <button type="button" id="sendTopComment" class="sendTopComment weButton u2i-button u2i-widget u2i-corner-all removeContentStyles">Send</button>' +
     '  </span>' +
-    '  <span id="signIn" class="gridSignInTop  removeContentStyles">&#129300</span>' +
+    '  <span id="signIn" class="gridSignInTop">&#129300</span>' +
     '  <span id="loginPopUp"></span>' +
     '  <div id="dlgAnchor"></div>' +
   '</div>';
   $('#topMenu').append(topMarkup);
 
   setSignInOutMarkup ();
+
+  // WV-224: Add event listener to the "Open Admin" button
+  $('#openAdminButton').click(async function () {
+    const state = await getGlobalState();
+    const { voterGuidePossibilityId} = state;
```

Comment:
Typically you would have a trailing space like this -- I would recommend fixing: 
const { voterGuidePossibilityId } = state;

Reply:
@DaleMcGrew - Thank you for catching these! The hardcoded url is left over from early, unsuccessful attempts that I thought I had removed. I will fix these items and resubmit the PR.

Reply:
Fixed!